src/actions/noting/create-note-at-caret.js

```diff
@@ -0,0 +1,39 @@
+var isVFocus = require('../../utils/vfocus/is-vfocus');
```

Comment:
Noticed that this function isn't actually used. Is that intentional and coming in the next PR?